src/scripts/main.js

```diff
@@ -1,3 +1,54 @@
 'use strict';
 
-// write code here
+const listMain = document.querySelector('ul');
+
+function convNumber(text) {
+  const transformTextPhase1 = text.split('');
+
+  transformTextPhase1.shift();
+
+  const transformTextPhase2 = transformTextPhase1.join('');
+  const transformTextPhase3 = transformTextPhase2.split(',').join('');
+  const result = Number(transformTextPhase3);
+
+  return result;
+}
+
+function sortList(list) {
+  const items = list.querySelectorAll('li');
+  const itemsArray = [ ...items ];
+
+  for (const elem of itemsArray) {
+    const first = elem.dataset.salary;
+
+    elem.dataset.salary = convNumber(first);
+  }
```

Comment:
you don't need to overwrite the value. just use the function `convNumber` directly in `.sort` method below